### ISSUE TITLE
Hammer commands for content export/import broken

### ISSUE THREAD

**Issue by maximiliankolb** [theforeman/foreman-documentation#1263]
Hi.

I think various Hammer commands are broken. I have tested the following command on Foreman 2.3, 2.5, 3.1, and 3.2:

```
# hammer content-export complete repository --organization-id 1 --product="My Product" --name="My Repo"
Error: No such sub-command 'repository'.

See: 'hammer content-export complete --help'.
```

source: guides/common/modules/proc_exporting-a-repository.adoc

**Comment by jeremylenz** [comment]
Repository exports were added in https://projects.theforeman.org/issues/34374 which is in Katello 4.5 / Foreman 3.3.

**Comment by maximiliankolb** [comment]
Fixed for 3.3+: https://github.com/theforeman/foreman-documentation/pull/1612
